### PR TITLE
MES-3926 - Vehicle checks question

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
## Description
- Added Show Me & Tell Me cards to the Vehicle Checks Modal
- Created a Vehicle Checks Question Component 
- Added ability to select, emit, disable and rehydrate the select field
- Updated to the latest schema version

More PR's will be done to:
- Add the radio buttons
- Connect it up with the store

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

Simulator Screen Shot - iPad Pro (10 5-inch) - 2019-11-04 at 13 41 50

### CODE REVIEW

File: src/pages/waiting-room-to-car/cat-be/components/vehicle-checks-modal/vehicle-checks-modal.cat-be.page.html [hunk 1]
```diff
@@ -1,16 +1,36 @@
 <ion-header>
-    <ion-navbar>
```
Comment (szabikr) [inline src/pages/waiting-room-to-car/cat-be/components/vehicle-checks-modal/vehicle-checks-modal.cat-be.page.html:2]: Leave the codebase in a better shape that you've found it. Nice one!

**Review by szabikr — APPROVED** [review]

File: src/environment/test-schema-version.ts [hunk 2]
```diff
@@ -1 +1 @@
-export const version = '3.2.3';
+export const version = '3.2.4';
```
Comment (szabikr) [inline src/environment/test-schema-version.ts:1]: Assuming this needs to be `3.2.5`

Comment (TomBillingtonUK) [reply src/environment/test-schema-version.ts:1]: That file shouldn't even be in github

**Review by rossfauldsbjss — APPROVED** [review]
Looks good.

**Pull request merged**